src/views/Favorite/FavoriteView.vue

```diff
@@ -1,12 +1,53 @@
 <template>
-    <p>Favoritos</p>
-  </template>
-  
-  <script lang="ts">
-  import { defineComponent } from 'vue';
-  
+  <section class="w-full justify-between pt-6">
+    <h1 class="text-white flex justify-center text-3xl">Minha Lista de Favoritos</h1>
+      <div>
+        <template v-if="favoriteMovies.length === 0 && favoriteSeries.length === 0">
+          <p class="text-center text-xl text-gray-500 pt-4"> Hum... parece que você ainda não tem favoritos.
+          <br>Explore nossos filmes e séries e adicione aqui os seus favoritos!</p>
+        </template>
+        <template v-else>
+          <div class="grid grid-cols-3 md:grid-cols-3 lg:grid-cols-4 gap-4 p-4">
+              <movies-card v-for="movie in favoriteMovies" :key="movie.id" :movie="movie"/>
```

Comment:
Set optional on this case. If you have an array which has no content the key id will be undefined.